conda/recipes/libcudf/recipe.yaml

```diff
@@ -138,6 +138,7 @@ outputs:
           - cuda-nvrtc
           - cuda-nvtx
           - cuda-version
+          - cudatoolkit
```

Comment:
Should this be conditional?
```suggestion
          - if: cuda_major == "11"
            then:
              - cudatoolkit
```

Reply:
Similarly, perhaps we should ignore exports from `cuda-cudart`, `cuda-nvrtc` for `cuda_major != "11"`.

Reply:
Probably, I'm going to check the logs of a previous run and check on which overpinned dependencies might cause trouble

Reply:
ignoring things that aren't in scope doesn't cause problems, so I can circle back to those when I have some followup fixes, but this should get things working

Reply:
but the `cudatoolkit` ignore is now conditional